MobileWallet/Common/Extensions/UIApplication+keyWindow.swift

```diff
@@ -0,0 +1,52 @@
+//  UIApplication+keyWindow.swift
```

Comment:
File names should also follow naming & case conventions:
`UIApplication+keyWindow.swift` => `UIApplication+KeyWindow.swift`